control-station/src/services/PodSocketClient.ts

```diff
@@ -64,24 +73,28 @@ class PodSocketClient {
 		this.socket.emit("load", (response: string) => {
 			console.log("Server acknowledged:", response);
 		});
+		this.setPodData((d) => ({ ...d, state: State.Load }));
```

Comment:
Issue [repeating]: the control station must not falsely claim the pod state was updated when it actually might not have been. We'll need to update `podData` only when a proper acknowledgement is received (within the callback).